src/pal/src/thread/context.cpp

```diff
@@ -478,7 +478,7 @@ void CONTEXTFromNativeContext(const native_context_t *native, LPCONTEXT lpContex
     }
 #undef ASSIGN_REG
     
-    if ((contextFlags & CONTEXT_FLOATING_POINT) == CONTEXT_FLOATING_POINT)
+    if ((contextFlags & CONTEXT_FLOATING_POINT) == CONTEXT_FLOATING_POINT && native->uc_mcontext.fpregs != nullptr)
```

Comment:
We should also clean the CONTEXT_FLOATING_POINT flag on the CONTEXT so that when the CONTEXT produced by this function is used somewhere or passed to the CONTEXTToNativeContext function, we don't use the garbage in the floating point registers that we've never filled in.